client-network/src/network_ops/default_network_ops_client.rs

```diff
@@ -137,20 +137,7 @@ where
         transactions: Vec<(TxoPointer, TxOut)>,
         to_address: StakedStateAddress,
         attributes: StakedStateOpAttributes,
-        verify_staking: bool,
     ) -> Result<(TxAux, TransactionPending)> {
-        // if the to_address belongs to current wallet, we do not check the state
-        let staking_addresses = self.wallet_client.staking_addresses(name, enckey)?;
-        if !staking_addresses.contains(&to_address) {
-            let staked_state = self.get_staked_state(name, &to_address, verify_staking)?;
-            verify_unjailed(&staked_state).map_err(|e| {
-                Error::new(
-                    ErrorKind::ValidationError,
-                    format!("Failed to validate staking account: {}", e),
-                )
-            })?;
```

Comment:
maybe the jailing test could be optionally run (if the state exists?)

Reply:
Done